package.json

```diff
@@ -1,20 +1,42 @@
 {
-  "private": true,
-  "scripts": {
-    "prepare": "yarn tslint && lerna run prepare",
-    "rebuild:browser": "theia rebuild:browser",
-    "rebuild:electron": "theia rebuild:electron",
-    "tslint-fix": "tslint --fix --project github-extension",
-    "tslint": "tslint --project github-extension"
-  },
+  "name": "@eclipse-che/theia-github-plugin",
+  "publisher": "theia",
+  "keywords": [
+    "theia-plugin"
+  ],
+  "version": "0.0.1",
   "license": "EPL-2.0",
+  "contributors": [
+    {
+      "name": "Igor Vinokur",
+      "email": "ivinokur@redhat.com"
+    }
+  ],
+  "files": [
+    "src"
+  ],
+  "dependencies": {
+    "axios": "0.18.0"
+  },
   "devDependencies": {
-    "lerna": "2.4.0",
-    "tslint": "5.10.0"
+    "@octokit/rest": "15.15.1",
+    "@theia/plugin": "0.3.16",
+    "@types/node": "10.12.1",
+    "rimraf": "2.6.2",
+    "tslint": "5.11.0",
+    "typescript": "2.9.2"
+  },
+  "scripts": {
+    "prepare": "yarn tslint && yarn run clean && yarn run build",
+    "clean": "rimraf lib",
+    "build": "tsc",
```

Comment:
I am not sure how you are testing it, but this project do not generate the plugin artifact .. like here: https://github.com/eclipse/che-theia-factory-extension/blob/master/package.json#L29

Reply:
fixed